Core/src/SplineInterpretation/SplineInterpretation.cpp

```diff
@@ -454,20 +444,14 @@ std::tuple<std::vector<int>, double> OpenInfraPlatform::UserInterface::SplineInt
 	return { indicator, curvatureZero };
 }
 
-double OpenInfraPlatform::UserInterface::SplineInterpretation::obtainThreshold(
-	const std::vector<std::pair<double, double>>& lengthsWithCurvatures) const throw(...)
-{
-	return obtainThreshold(obtainCurvatureFrom_lengthWithCurvatures(lengthsWithCurvatures));
-}
-
-double OpenInfraPlatform::UserInterface::SplineInterpretation::obtainThreshold(std::vector<double> data) const throw(...) {
+double OpenInfraPlatform::Core::SplineInterpretation::SplineInterpretation::obtainThreshold(std::vector<double> data) const throw(...) {

```

Comment:
Beginning with vs2019 (which we'll be switching to soon) the `throw(...)` should be replaced with `noexcept(false)`. Please replace in all new files written by you accordingly.

Reply:
Done in commits 94afcd3 and 615d2b1